### DISCUSSION TITLE
Version 1.25.0

### DISCUSSION THREAD

**Discussion by apple502j** [ScratchAddons/ScratchAddons#4306] [Changelog]
# Bugfixes
- [#4121](https://github.com/ScratchAddons/ScratchAddons/issues/4121) customize-avatar-border does not affect avatar outlines added by scratchr2. (prio-4)
- [#4234](https://github.com/ScratchAddons/ScratchAddons/issues/4234) scratch-messaging shows Delete button on comments that cannot be deleted. (prio-4)
- [#4250](https://github.com/ScratchAddons/ScratchAddons/issues/4250) dark-www and scratchr2 does not affect class reporting modal. (prio-4)
- [#4254](https://github.com/ScratchAddons/ScratchAddons/issues/4254) dark-www loads removed userstyle. (prio-4)
- [#4264](https://github.com/ScratchAddons/ScratchAddons/issues/4264) scratchr2 causes My Ocular status to overflow. (prio-4)
- [#4266](https://github.com/ScratchAddons/ScratchAddons/issues/4266) gamepad does not reset controls properly when loading a new project. (prio-3)
- [#4279](https://github.com/ScratchAddons/ScratchAddons/issues/4279) hide-new-variables causes significant amount of lag. (prio-2)
- [#4284](https://github.com/ScratchAddons/ScratchAddons/issues/4284) debugger single stepping feature alters the behavior of broadcast blocks. (prio-3)
- [#4284](https://github.com/ScratchAddons/ScratchAddons/issues/4284) debugger breakpoint block does not work in some cases. (prio-3)
- [#4289](https://github.com/ScratchAddons/ScratchAddons/issues/4289) dark-www uses wrong banner color after disabling or enabling scratchr2. (prio-4)
- [#4295](https://github.com/ScratchAddons/ScratchAddons/issues/4295) scratch-messaging crashes while handling certain replies. (prio-2)
- [#4302](https://github.com/ScratchAddons/ScratchAddons/issues/4302) forum-toolbar does not work on signature settings page. (prio-4)
- [#4314](https://github.com/ScratchAddons/ScratchAddons/issues/4314) scratchstats uses wrong timestamp in graph label. (prio-4)
- [#4314](https://github.com/ScratchAddons/ScratchAddons/issues/4314) scratchstats uses wrong step size in graph Y axis. (prio-4)
- [#4319](https://github.com/ScratchAddons/ScratchAddons/issues/4319) custom-zoom settings description contradicts the actual minimum value. (prio-4)
- [#4320](https://github.com/ScratchAddons/ScratchAddons/issues/4320) mute-project does not allow Cmd key to be used on macOS. (prio-3)
- [#4321](https://github.com/ScratchAddons/ScratchAddons/issues/4321) Comment checker does not filter URLs including "scratch-addons". (prio-3)
- [#4326](https://github.com/ScratchAddons/ScratchAddons/issues/4326) scratchr2 uses wrong width on some elements in Firefox. (prio-4)
- [#4339](https://github.com/ScratchAddons/ScratchAddons/issues/4339) Settings buttons do not use the proper cursor when hovered. (prio-4)
- [#4340](https://github.com/ScratchAddons/ScratchAddons/issues/4340) Disabling all addons causes the banner on the popup settings to disappear. (prio-4)
- [#4340](https://github.com/ScratchAddons/ScratchAddons/issues/4340) Empty search result on the settings causes it to display blank page. (prio-3)
- [#4343](https://github.com/ScratchAddons/ScratchAddons/issues/4343) scratchr2 causes forum post's username section to render incorrectly. (prio-4)
- [#4346](https://github.com/ScratchAddons/ScratchAddons/issues/4346) dark-www does not apply to the new Become a Scratcher page. (prio-4)

# New Features
- [#4226](https://github.com/ScratchAddons/ScratchAddons/issues/4226) New addon: horizontal-mystuff-tabs (prio-3)
- [#4253](https://github.com/ScratchAddons/ScratchAddons/issues/4253) New addon: forum-live-preview (prio-3)
- [#4256](https://github.com/ScratchAddons/ScratchAddons/issues/4256) msg-count-badge: add Orange preset (prio-3)

## Other Changes
- [#4335](https://github.com/ScratchAddons/ScratchAddons/issues/4335) Increase the minimum supported version for Firefox to 86 (prio-3)
- [#4341](https://github.com/ScratchAddons/ScratchAddons/issues/4341) Settings: addon categories can now be closed by clicking it again. (prio-4)

**Comment by sakhiy2010** [comment]
When it’s released?

**Reply by WorldLanguages** [reply]
In the following days
